pkg/cas/read_write_decoupling_content_addressable_storage.go

```diff
@@ -28,41 +29,71 @@ func NewReadWriteDecouplingContentAddressableStorage(reader cas.ContentAddressab
 }
 
 func (cas *readWriteDecouplingContentAddressableStorage) GetAction(ctx context.Context, digest *util.Digest) (*remoteexecution.Action, error) {
+	ctx, span := trace.StartSpan(ctx, "cas.RWDecoupling.GetAction")
```

Comment:
The same here: should we add a TracingContentAddressableStorage that acts as a decorator?